server/server.go

```diff
@@ -888,6 +868,22 @@ func (server *Server) loadConfig(configurations configs, globalConfiguration Glo
 	return serverEntryPoints, nil
 }
 
+func configureLBServers(lb healthcheck.LoadBalancer, config *types.Configuration, frontend *types.Frontend) error {
+	for serverName, server := range config.Backends[frontend.Backend].Servers {
+		url, err := url.Parse(server.URL)
+		if err != nil {
+			log.Errorf("Error parsing server URL %s: %v", server.URL, err)
+			return err
+		}
+		log.Debugf("Creating server %s at %s with weight %d", serverName, url.String(), server.Weight)
```

Comment:
>rename url to URL

This may give readers the false impression that `URL` is an exported identifier. Now you basically have to track down the scope of the variable and realize that it's defined on the stack. 🤔